pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1158,6 +1161,14 @@ func getLoopbackAdrress(proto iptables.Protocol) string {
 	}
 }
 
+func portsUsedByLiveMigration() []int {
+	return []int{
+		LibvirtLocalConnectionPort,
+		migrationproxy.LibvirtDirectMigrationPort,
+		migrationproxy.LibvirtBlockMigrationPort,
+	}
+}
+
```

Comment:
I'm not sure if we want to define the constants at global scope. Because they are currently defined within Describe block, I can't move this func outside.

Reply:
? Describe block ? This is a production code file. I don't understand.

Reply:
Addressed in a new [commit](https://github.com/kubevirt/kubevirt/pull/5514/commits/0bfca3ed25b22948fe3b703459b3419fcf787730)